README.md

```diff
@@ -223,238 +260,369 @@ To enable this feature:
+  (environment variable -- The timeout multiplier for archive status updates. If the first
+  archive status update polling fails, subsequent polling intervals will apply this multiplier
+  successively. Set to a lower number to poll more often. The default value is 1.5.
 
-- `ENABLE_ARCHIVE_MANAGER`: (Optional, default value: false) Enable Archive Manager. Only meaningful if `archiving` is not disabled (Manage Recordings, requires firebase to be configured)
-- `EMPTY_ROOM_LIFETIME`: (Optional, default value 3): Maximum time, in minutes, an empty room
+* `Archiving.archivingManager.enabled` (config.json) / `ENABLE_ARCHIVE_MANAGER`
+  (environment variable) -- Whether the UI will include a list of archive recordings for the session.
+  The end user can click an archive to view the recording. This setting is only meaningful
+  if `Archiving.enabled` / `ENABLE_ARCHIVING` is set to `true`. The default value is `false`.
+
+* `Archiving.archivingManager.emptyRoomMaxLifetime` (config.json) / `EMPTY_ROOM_LIFETIME`
+  (environment variable) -- The time, in minutes, an empty room will store its archive history
+  after the last participant leaves a room, in minutes. The default value is 3.
+
+
+**Config.json example:**
 
 ```json
-"Archiving": {
-    "enabled": true,
-    "archiveAlways": false,
-    "pollingInitialTimeout": 5000,
-    "pollingTimeoutMultiplier": 1.5,
-    "archiveManager": {
-        "enabled": false,
-        "emptyRoomMaxLifetime": 3
+{
+    "Archiving": {
+        "enabled": true,
+        "archiveAlways": true,
+        "archiveAlways": 10,
+        "pollingInitialTimeout": 10000,
+        "pollingTimeoutMultiplier": 1,
+        "archiveManager": {
+            "enabled": true,
+            "emptyRoomMaxLifetime": 10
+        }
     }
-},
+}
+```
+
+**Environment variable example:**
+
+```sh
+export GOOGLE_CLIENT_ID=yourClientId.apps.googleusercontent.com;
+export GOOGLE_HOSTED_DOMAIN=yourhosteddomain.com;
+export SIP_REQUIRE_GOOGLE_AUTH=true;
 ```
 
-#### Screen sharing configuration
+### Screen sharing
 
-- `ENABLE_SCREENSHARING`:(Optional, default value: false) Whether to enable screen sharing.
-- `CHROME_EXTENSION_ID` :(Optional, LEGACY, default value: 'null'): The Chrome AddOn extension ID for screen sharing. Note: the Chrome extension is no longer required for screensharing.
-- `ENABLE_ANNOTATIONS`: (Optional, default value: true) Whether to enable annotations in screen sharing. This is only meaningful if screen sharing is not disabled.
+To enable and configure screen sharing, edit options in the config/config.json file
+or set environment variables:
+
+* `Screensharing.enabled` (config.json) / `ENABLE_SCREENSHARING` (environment variable) -- Whether
+  to enable screen sharing. The default value is `false`.
+
+* `Screensharing.chromeExtensionId` (config.json) / `CHROME_EXTENSION_ID`
+  (environment variable) -- The Chrome add-on extension ID for screen sharing.
```

Comment:
I don't think this is needed anymore. Chrome now supports screensharing without it. The only relevant parameter is ENABLE_SCREENSHARING. I think.

Reply:
A screen-sharing extension is required by older versions of Chrome and Opera (which we still support). I'll make the README docs clearer.